tests/api/130_ProcessingCest.php

```diff
@@ -38,15 +53,88 @@ class ProcessingCest
         'updated_at'            => 'string',
     ];
 
+    public function create_processing_test(\ApiTester $I)
+    {
+        $I->amGoingTo('Create a new Processing');
+        $I->login();
+
+        $this->processingData['folder_id'] = $I->getRootFolderId();
+
+        $I->sendJsonToCreate(ProcessingCest::ROUTE, $this->processingData);
+
+        $I->seeCorrectJsonResponse($this->processingJsonType);
+        $I->seeResponseContainsJson([
+            'name' => $this->processingData['name'],
+        ]);
+
+        $this->processing = json_decode(json_encode($I->getPreviousResponse()), JSON_OBJECT_AS_ARRAY);
+    }
+
+    /**
+     * @depends create_processing_test
+     */
     public function list_processings_test(\ApiTester $I)
     {
         $I->amGoingTo('List available Processings');
 
         $I->login();
 
-        $I->sendGET('/processings');
+        $I->sendGET(ProcessingCest::ROUTE);
```

Comment:
Why not simply `self::ROUTE` ?

Reply:
Because we don't necessarily know what 'self' is outside the context. The class name is more explicit.

Reply:
Ok that explain the explicit visibility on the constant declaration.